src/features/connections/components/IconLink.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image"
+
+export type IconLinkProps = {
+	url: string,
+	icon: string,
+	label: string,
+	reverse?: boolean
+}
+
+const IconLink = ({ url, icon, label, reverse = false }: IconLinkProps) => {
+
+	return (
+		<div className="odd:bg-slate-900 even:bg-slate-800
+		flex even:flex-row sm:even:flex-row-reverse lg:even:flex-row">
+			<a
+				href={url}
+				target='blank'
+				className="items-center py-5 w-96
+				text-left pl-5 sm:pl-10
+				flex justify-start"
+			>
+				<div className="w-32 h-32">
+					<Image src={icon} alt="slack logo" />
```

Comment:
The NextJS `Image` component will throw an error if you don't provide a `height` and `width` attribute ([with some exceptions](https://nextjs.org/docs/api-reference/next/image)).

You could solve this one of three ways:

- If you know what size it should always be, hard-code the height and width.

- If you want to pass the decision to whoever uses the component, add `height` and `width` to the component props.

- If you want to allow the consumer of the component total freedom over the image, including the options to use `next/image` advanced features that don't require `height` and `width`, or to not use `next/image` at all in favor of a plain `<img>` element, you can accept the whole thing as an `imageComponent` prop. (Tip: To find the TS type of an element or component, you can hover over it and copy/paste it from the tooltip that pops up!)

That last one borders on unnecessary complexity... but you can stretch those muscles if you're feeling it!